mmv1/third_party/terraform/website/docs/r/compute_router_peer.html.markdown

```diff
@@ -361,6 +382,17 @@ The following arguments are supported:
   BFD declares that a peer is unavailable. If set, the value must
   be a value between 5 and 16.
 
+<a name="nested_md5_authentication_key"></a>The `md5_authentication_key` block supports:
+
+* `name` -
+  (Required)
+  Name used to identify the key. Must be unique within a router. Must be
+  referenced by exactly one bgpPeer. Must comply with RFC1035.
+
+* `key` -
+  (Required, Input Only)
+  Value of the key. Maximum length is 80 characters. Can only contain printable ASCII characters
```

Comment:
nit: Similar to above, this reads like a proto comment more than a user-facing docstring. Maybe steal from https://cloud.google.com/sdk/gcloud/reference/compute/routers/add-bgp-peer#--md5-authentication-key for this one? `The MD5 authentication key for this BGP peer. Maximum length is 80 characters. Can contain only printable ASCII characters.`

Reply:
Ack.